app/src/home.js

```diff
@@ -242,7 +244,79 @@ export default function Home() {
     });
   }, [ ])
 
+  const handleSave = (e) => {
+    e.preventDefault()
+    saveSession()
+  }
+
+  async function createSession(data) {
+    const apiName = 'MTADemoAPI';
+    const path = 'createSession';
+    const myInit = { 
+    //   headers: { 
+    //     Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}`,
+    //   },
+      response: true,
+      queryStringParameters: data
+    };
+
+    const result =  await API.post(apiName, path, myInit); 
+
+    return result;
+  }
 
+  const saveSession = () => {
+    const id = uuid();
+    Storage.configure({
+      bucket: 'mtastack-mtastackstorages3bucketc161f3b3-1tfncqzctldb1',
+      level: 'public',
+      region: 'us-west-2',
+    });
+    const transcribeAddress = `${id}/transrcibe-medical-output/transcribe.txt`
```

Comment:
I believe we decided to store in the following format during design review

        *  bucket/transcribe-medical-output/<sessionid>/ sessionid-session-transcription.txt
        * bucket/comprehend-medical-output/<sessionid>/ sessionid-session-comprehend.tx

Reply:
Modified. https://github.com/aws-samples/medical-transcription-analysis/blob/eafae50489bbda5b2cd751d3679ef7a7ae8d11ce/app/src/home.js#L279-L280